components/HeartBitBanner.tsx

```diff
@@ -41,7 +41,7 @@ export default function HeartBitBanner() {
               'text-xl': !isMediaMax1025px,
             })}
           >
-            Agent SDK - Explore Multi Agent Collaboration Onchain
+            Ditch Google Docs. Explore Onchain dDocs!
```

Comment:
Are we not using * here in Google Docs?

Reply:
This was the earlier copy, let me confirm with Laszlo

Reply:
Deploy once you get the confirmation!